src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -269,7 +270,11 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         /// If both <see cref="IssuerSigningKeyValidatorUsingConfiguration"/> and <see cref="IssuerSigningKeyValidator"/> are set, IssuerSigningKeyResolverUsingConfiguration takes
         /// priority.
         /// </remarks>
-        public IssuerSigningKeyValidator IssuerSigningKeyValidator { get; set; }
+        public IssuerSigningKeyValidatorDelegate IssuerSigningKeyValidator
+        {
+            get => _issuerSigningKeyValidator;
+            set => _issuerSigningKeyValidator = value ?? throw new ArgumentNullException(nameof(value), "IssuerSigningKeyValidator cannot be set as null.");
```

Comment:
Consider creating a [log message](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dac7a56d0b5403980a5d1eb7afaa477b642e4c4f/src/Microsoft.IdentityModel.Tokens/LogMessages.cs#L12).

Reply:
We have a number of similar scenarios. I will definitely look into generalising them and reusing a log message or creating a new one as part of this stream.